components/profile/ProfileScreenBase.tsx

```
@@ -49,6 +50,7 @@ const ProfileScreenBase: React.FC<ProfileScreenBaseProps> = function ProfileScre
 }) {
   const { t } = useTranslation();
   const { triggerSync } = useSyncContext();
+  const { t } = useI18n();
```

Biome (1.9.4)

[error] Shouldn't redeclare 't'. Consider to delete it or rename it.

't' is defined here:


(lint/suspicious/noRedeclare)